### ISSUE TITLE
'the identity' vs 'an identity'

### ISSUE THREAD

**Issue by dig-comments** [usnistgov/800-63-3#860]
**Organization Name (N/A, if individual)**: National Security Agency

**Organization Type**: 1
 
**Document (63-3, 63A, 63B, or 63C)**: 800-63B

**Reference (Include section and paragraph number)**: 6.1, 2
 
**Comment (Include rationale for comment)**:  Suggest changing 'the identity' to 'an identity' in the first sentence, which makes this apply to all identities and not just one in particular. 
 
Also, the second sentence is a bit confusing. Is the CSP suppose to keep the information on how he configured the throttling of attempts or is the CSP suppose to keep information on the number of attempts and their frequency for a particular authentication of an identity? Suggest rewording to: "The CSP or verifier SHALL also maintain the information required for the number of authentication attempts on a particular account or identity, as described in section 5.2.2 Rate Limiting (Throttling)."

Original text: "Throughout the online identity lifecycle, CSPs SHALL maintain a record of all authenticators that are or have been associated with the identity. The CSP or verifier SHALL also maintain the information required for throttling authentication attempts when required, as described in section 5.2.2."

**Suggested Change**: Make suggested change.
 
---
 
Organization Type: 1 = Federal, 2 = Industry, 3 = Academia, 4 = Self, 5 = Other

**Comment by jimfenton** [comment]
Changing instead to "each identity" since this record needs to be kept individually and not in the aggregate.
